toolchain/sem_ir/file.h

```diff
@@ -62,6 +62,9 @@ struct Class : public Printable<Class> {
     out << "}";
   }
 
+  // Determine whether this class has been fully defined.
```

Comment:
You might comment here that this is false when definition_id is valid while the definition is being processed (maybe holistically document the transition states for members that start as invalid?)

Reply:
Expanded the comments to say when each member is set, and to clarify that this determines whether we've reached the `}`.